Tools/GNUMake/comps/nvcc.mak

```diff
@@ -11,6 +11,13 @@ ifeq ($(nvcc_major_lt_8),1)
   $(error Your nvcc version is $(nvcc_version). This is unsupported. Please use CUDA toolkit version 8.0 or newer.)
 endif
 
+nvcc_forward_unknowns = 0
+ifeq ($(shell expr $(nvcc_major_version) \>= 10),1)
```

Comment:
I think this check needs to be fixed so it's correct for 11.0+.

Reply:
You are absolutely right.